lib/api/request_models/accepte_invitattion_response.dart

```diff
@@ -0,0 +1,7 @@
+import 'package:client_common/api/response_models/api_response.dart';
```

Comment:
rename this file

"accepte_invitattion_response" -> "accept_invitation_response"